.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Report a problem
+title: "[Bug] "
+labels: 'bug'
+assignees: ''
+
+---
+### Description
+
+### Steps to Reproduce
+
+1.
+2.
+3.
+
+### Expected Behavior
+<!-- What did you expect to happen when you did the above? -->
+
+### Actual Behavior
+<!-- What actually happened when you did the above? -->
+
+### Basic Information
+
+- code42cli version: <!-- get the version using `pip show code42cli`-->
+- python version: <!-- get the version using `python --version`-->
```

Comment:
Since just running `python --version` isn't guaranteed to actually be what the cli was installed under, I wonder if we should have `code42 -h` also print the python version, then we could get accurate cli version, py42 version, and python version just by having them do `code42 -h` for us.

Reply:
good call, I like that